server/channels/app/platform/web_hub.go

```diff
@@ -693,6 +790,20 @@ func (i *hubConnectionIndex) ForUser(id string) []*WebConn {
 	return conns
 }
 
+// ForChannel returns all connections for a channelID.
+func (i *hubConnectionIndex) ForChannel(channelID string) []*WebConn {
+	if len(i.byChannelID[channelID]) <= 1 {
+		return i.byChannelID[channelID]
+	}
```

Comment:
Wondering if the fast path here is adding much value. It feels like additional code to cover an unrealistic case that would be fast regardless. Thoughts?

Reply:
Makes sense. Removing.